src/iap.ts

```diff
@@ -273,13 +253,15 @@ export const getAvailablePurchases = (): Promise<
  * @param {string} [obfuscatedProfileIdAndroid] Specifies an optional obfuscated string that is uniquely associated with the user's profile in your app.
  * @returns {Promise<InAppPurchase>}
  */
-export const requestPurchase = ({
-  sku,
-  andDangerouslyFinishTransactionAutomaticallyIOS = false,
-  applicationUsername,
-  obfuscatedAccountIdAndroid = undefined,
-  obfuscatedProfileIdAndroid = undefined,
-}: RequestPurchase): Promise<InAppPurchase> =>
+
+export const requestPurchase = (
+  sku: string,
+  appAccountToken: string,
+  andDangerouslyFinishTransactionAutomaticallyIOS: boolean = false,
+  obfuscatedAccountIdAndroid: string | undefined = undefined,
+  obfuscatedProfileIdAndroid: string | undefined = undefined,
+  applicationUsername: string | undefined = undefined,
+): Promise<InAppPurchase> =>
```

Comment:
This breaks my change with passing configuration object to this function from this PR https://github.com/dooboolab/react-native-iap/pull/1766... Also `appAccountToken` shouldn't be required param.